cli/di.container.ts

```diff
@@ -176,9 +179,11 @@ export default function configureContainer(commandName: CommandName, cliArgs: an
     getTransactionReceipt: asFunction(provideGetTransactionReceipt),
 
     getTraceData: asFunction(provideGetTraceData),
+    getAgentLogs: asFunction(provideGetAgentLogs),
     traceRpcUrl: asFunction((fortaConfig: FortaConfig) => {
       return fortaConfig.traceRpcUrl
     }).singleton(),
+    fortaApiUrl: asValue('https://api.forta.network'),
```

Comment:
❓Wasn't sure if this should be added to the `FortaConfig` interface or a new interface?